pkg/utils/data.go

```diff
@@ -35,14 +35,14 @@ type AgentInfo struct {
 // ProbeResult structure for network probing results
 type ProbeResult struct {
 	URL              string
-	ConnectionResult int
-	HTTPCode         int
-	Total            int
-	ContentTransfer  int
-	TCPConnection    int
-	DNSLookup        int
-	Connect          int
-	ServerProcessing int
+	ConnectionResult float64
+	HTTPCode         float64
+	Total            float64
+	ContentTransfer  float64
+	TCPConnection    float64
+	DNSLookup        float64
+	Connect          float64
+	ServerProcessing float64
```

Comment:
No! :) Let's convert the type when using it in gauge. Now it looks inconsistent that data comes as integer (in http request and in agent data structure) and it is converted into float during deserialization. It's better to have ProbeResult the same as in agent.